core/shared/src/main/scala/scalaz/zio/Fiber.scala

```diff
@@ -31,12 +31,12 @@ trait Fiber[E, A] { self =>
   def join: IO[E, A]
 
   /**
-   * Interrupts the fiber with the specified error. If the fiber has already
+   * Interrupts the fiber with the specified error(s). If the fiber has already
    * terminated, either successfully or with error, this will resume
    * immediately. Otherwise, it will resume when the fiber has been
    * successfully interrupted or has produced its result.
    */
-  def interrupt[E2](t: Throwable): IO[E2, Unit]
+  def interrupt[E2](ts: Throwable*): IO[E2, Unit]
```

Comment:
Can we do two variants:

```scala
def interrupt: IO[E2, Unit] = interrupt0(Nil)

def interrupt(t: Throwable, ts: Throwable*): IO[E2, Unit] = interrupt0(t :: ts.toList)

protected def interrupt0[E2](ts: List[Throwable]): IO[E2, Unit]
```

This will allow a cleaner syntax for the case of no concrete cause.

Reply:
Oh yeah, I didn't think about that 👌